features/bootstrap/FeatureContext.php

```diff
@@ -244,6 +318,14 @@ public function requestOnWithResolveOptionShouldContainFollowingHeaders($host, $
 		}
 	}
 
+	/**
+	 * @BeforeScenario
+	 */
+	public function cleanupCommands(BeforeScenarioScope $scope)
+	{
+		$this->commands = [];
+	}
+
 	/**
 	 * @AfterFeature
 	 */
```

Comment:
Add the sites of this feature in `cleanup`.

Reply:
Yes. Didn't add it before as updated cleanup function was in another PR(which was yet to be merged)